src/sugar.jl

```diff
@@ -133,7 +133,7 @@ struct _UpdateOp{OP,V}
 end
 (u::_UpdateOp)(x) = u.op(x, u.val)
 
-function atset_impl(ex::Expr; overwrite::Bool)
+function atset_impl(ex::Expr; overwrite::Bool=false, lenstransform=identity)
```

Comment:
Let's rename these functions. How about `lensmacro`/`setmacro`? Also a docstring should be added.

Reply:
`setmacro` sounds good.